diff --git a/conf/application.conf b/conf/application.conf
index 25a6062..7416583 100644
--- a/conf/application.conf
+++ b/conf/application.conf
@@ -2,6 +2,7 @@
 play.editor="http://localhost:63342/api/file/?file=%s&line=%s"
 
 play.http.secret.key="whatever"
+play.http.secret.key=${?APPLICATION_SECRET}
 play.modules.enabled += "play.modules.swagger.SwaggerModule"
 play.filters.headers.contentSecurityPolicy = null
 play.filters.disabled += play.filters.csrf.CSRFFilter
